src/main/xar-resources/data/jmx/jmx.xml

```diff
@@ -167,11 +153,23 @@
             <para>For example, to get a report on current memory usage and running instances, use
                 the following URL:</para>
             <programlisting>http://localhost:8080/exist/status?c=memory&amp;c=instances</programlisting>
+            <note>
+                <para>The JMX information is only accessible via the HTTP interface when either:</para>
+                <itemizedlist>
+                    <listitem>
+                        <para>the client accesses the API on localhost</para>
+                    </listitem>
+                    <listitem>
+                        <para>a valid token is provided when accessing the API from a remote IP address. A file containing this token and an example of its use can be found in the eXist data directory at <code>$(EXIST_HOME)/data/jmxservlet.token</code>.</para>
```

Comment:
Can you please change `$(EXIST_HOME)` -> `$EXIST_HOME`? I think that will make it consistent with the other existing documentation